main.py

```diff
@@ -32,28 +31,42 @@
     default=undefined if "api_key=" not in REDASH_API_QUERY_URL else None,
 )
 
-DEFAULT_EXCLUDE_SOURCES = config(
+REDASH_TIMEOUT_SECONDS = config("REDASH_TIMEOUT_SECONDS", cast=int, default=60)
+
+EXCLUDE_SOURCES = config(
     "EXCLUDE_SOURCES",
     cast=Csv(),
     default="shield-recipe-client/*, normandy/*, main/url-classifier-skip-urls",
 )
 
 # XXX What about instead just say that `bad == status.endswith('_error')`??
-DEFAULT_GOOD_STATUSES = config(
-    "GOOD_STATUSES", cast=Csv(), default="success, up_to_date"
-)
+GOOD_STATUSES = config("GOOD_STATUSES", cast=Csv(), default="success, up_to_date")
 
 # XXX What about 'backoff' ??
-DEFAULT_NEUTRAL_STATUSES = config(
-    "NEUTRAL_STATUSES", cast=Csv(), default="pref_disabled"
-)
+NEUTRAL_STATUSES = config("NEUTRAL_STATUSES", cast=Csv(), default="pref_disabled")
 
 # Statuses to ignore if their total good+bad numbers are less than this.
-DEFAULT_MIN_TOTAL_ENTRIES = config("MIN_TOTAL_ENTRIES", cast=int, default=0)
+MIN_TOTAL_ENTRIES = config("MIN_TOTAL_ENTRIES", cast=int, default=1000)
 
 DEFAULT_ERROR_THRESHOLD_PERCENT = config(
     "DEFAULT_ERROR_THRESHOLD_PERCENT", cast=float, default=2.0
 )
+
+
+def _parse_threshold_percent(s):
+    name, percentage = s.split("=")
+    return (name.strip(), float(percentage))
+
+
+SPECIFIC_ERROR_THRESHOLD_PERCENT = dict(
+    config(
+        "SPECIFIC_ERROR_THRESHOLD_PERCENT",
+        cast=Csv(cast=_parse_threshold_percent, delimiter=";"),
+        default=("main/collection = 10; foo/bar = 2 "),
```

Comment:
What means default here? This looks more like a documentation example :)

Reply:
Good point! I liked that it was there to demonstrate the format but you're right it should only be our/Mozilla real defaults that we actually want to use. 

I think I'll remove it and put this into a code comment instead.